### ISSUE TITLE
Carthage support? :) 

### ISSUE THREAD

**Issue by Sajjon** [groue/CombineExpectations#1]
Hey! This project looks very nice indeed! Im the OP of the [Swift Forum thread you recently announced this project](https://forums.swift.org/t/combine-blocking-wait/27636/4) in.

I'm of course keen on using SPM, but unfortunately one of my dependencies [BitcoinKit does not yet support SPM](https://github.com/yenom/BitcoinKit/issues/224), so for now I'm stuck in Carthage land.

In a [fork of this project](https://github.com/Sajjon/CombineExpectations/tree/carthage) I'm trying adding a shared scheme, to a `swift package generate-xcodeproj` generated project, but when I try using this fork, carthage fails to build, due to XCTest:

```
ld: warning: Could not find or use auto-linked framework 'XCTest'
Undefined symbols for architecture arm64:
  "_OBJC_CLASS_$_XCTestExpectation", referenced from:
      objc-class-ref in PublisherExpectation.o
      objc-class-ref in Recorder.o
ld: symbol(s) not found for architecture arm64
```

Any clues on how to fix it?

**Comment by groue** [comment]
Hello @Sajjon,

Thanks for checking out this library! I guess you'll have to tell Carthage that XCTest should not be linked to the framework it produces. I'm not a Carthage user myself, so I haven't any clue about how to do that. You'll surely find better support on StackOverflow or in the Carthage repo itself: https://github.com/Carthage/Carthage/issues.

**Comment by Sajjon** [comment]
Thanks for your prompt reply @groue , yes I tried that but failed :/ Im no Carthage expert either. 

For now I copied your files manually into my project, and also prepended each file with your LICENCSE and also added a url linking to this github project. I think your MIT license allows for that, right? 

Not a perfect solution at all, but works for now, I don't have more time to spend on messing with carthage right now. And hopefully I can drop Carthage soon and migrate fully to SPM :)
